examples/check.rs

```diff
@@ -45,10 +45,26 @@ struct Args {
     all_targets: bool,
 }
 
+/// [`Command`] for launching cargo-plumbing binary
+///
+/// The `CARGO_PLUMBING_BIN` env variable should point to a `cargo-plumbing` binary. It is used
+/// mostly for testing examples.
+///
+/// See <https://github.com/crate-ci/cargo-plumbing/issues/108>
+fn cargo_plumbing_bin() -> Command {
```

Comment:
Can you move this towards the bottom?  Generally when people read source, they want an overview and then to decide from there what details to dig into.  Putting details at the top distracts from that and puts a burden on readers to figure out how this all fits together.